client/app.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import axios from 'axios';
+
+import GameDescription from './components/GameDescription.jsx'
+import Carousel from './components/Carousel.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      gameId: '1',
+      name: null,
+      description: null,
+      videoURL: null,
+      mainImageURL: null,
+      logoURL: null,
+      carouselImagesURL: [],
+      recentReviews: null,
+      allReviews: null,
+      releaseDate: null,
+      developer: null,
+      publisher: null,
+      tags: []
+    }
+    this.fetchGame = this.fetchGame.bind(this);
+  }
+
+  getRandomInt() {
+    return Math.floor(Math.random() * (100)) + 1;
+  }
+
+  fetchGame(gameId) {
+    axios.get('http://localhost:3001/' + gameId)
+      .then((data) => {
+        const gameData = data.data[0];
+        let carouselArray = this.state.carouselImagesURL.concat(gameData.carouselImagesURL);
+
+        for (var i = 0; i < 15; i++) {
+          carouselArray.push(gameData.carouselImagesURL);
+        };
```

Comment:
Looks like you're using an index counting up to 15 but then not doing anything with the index? Is the same image supposed to be pushed into the list every time?

Also why 15 specifically?

Reply:
This is just to feed it test data. This code will be removed when the database array will have more than one item